config/application.rb

```diff
@@ -1,6 +1,7 @@
 require File.expand_path('../boot', __FILE__)
 
 require 'rails/all'
+require 'dotenv' ; Dotenv.load '.env.development.env.local', ".env.#{Rails.env}"
```

Comment:
Why do we need it for different environments? One local `.env` file should be enough. In staging and production, they won't be used anyways.